ruff.toml

```diff
@@ -37,8 +37,8 @@ target-version = "py312"
 
 [lint]
 # Currently only enabled for F541 and I: https://docs.astral.sh/ruff/rules/
-select = ["F541", "I"]
-ignore = []
+select = ["F", "I"]
+ignore = ["F841", "F401", "F405", "F403"]
```

Comment:
My preferred way of tackling this initiative; then we can have separate tickets for F841, etc. or multiple PR's

Reply:
These 4 doesn't have autofixes?